python/pyspark/sql/column.py

```diff
@@ -151,6 +162,8 @@ def __init__(self, jc):
     __rdiv__ = _reverse_op("divide")
     __rtruediv__ = _reverse_op("divide")
     __rmod__ = _reverse_op("mod")
+    __pow__ = _bin_func_op("pow")
```

Comment:
I can rename, but what if later we would implement something like `__lshift__` or `__rshift__` to allow the syntax `df.a << 2`, then we would have to either rename `_pow` back to `_bin_func_op` and utilize it, or add one more functions. Now its clear that `_bin_func_op` allows you to utilize binary function. What do you think?